WooCommerce/Classes/Extensions/NumberFormatter+LocalizedOrNinetyNinePlus.swift

```diff
@@ -15,7 +15,8 @@ extension NumberFormatter {
     private enum Constants {
         static let ninetyNinePlus = NSLocalizedString(
             "99+",
-            comment: "Shown when there are more than 99 items of something (e.g. Processing Orders)."
+            comment: "Please limit to 3 characters if possible. This is used if " +
+                "there are more than 99 items like in the Orders tab."
```

Comment:
While I think it's correct, that "like" made the sentence a bit hard to parse for me. Maybe we can drop it if this is only used in orders, or switch it around: "...if there are more than 99 items in a tab, like Orders"

Reply:
Thank you for being thorough, @koke. 🙇 I used your second suggestion. 404547878772e8e4416197447867927537ac6282